jbehave-support-core/src/main/java/org/jbehavesupport/core/internal/expression/temporal/RandomDateTimeCommand.java

```diff
@@ -0,0 +1,21 @@
+package org.jbehavesupport.core.internal.expression.temporal;
+
+
+import static org.springframework.util.Assert.isTrue;
+
+import org.jbehavesupport.core.expression.ExpressionCommand;
+import org.jbehavesupport.core.internal.RandomGeneratorHelper;
+import org.springframework.stereotype.Component;
+
+/**
+ * Generate random {@link java.time.LocalDateTime}
+ */
+@Component
+public class RandomDateTimeCommand implements ExpressionCommand {
```

Comment:
it should be mentioned in Expression-commands.md

Reply:
Added, thanks for noticing.